client/src/components/PickupList.js

```diff
@@ -40,13 +41,15 @@ const PickupListPage: () => Node = () => {
     if(loading) return <tr><td>Loading...</td></tr>;
     if(error) return <tr><td>Loading...</td></tr>;
 
-    return data.appointments.map( ({id, startTime, endTime, location}) => (
+    return (
```

Comment:
since a list of `<tr>` can't be rendered without a `<table>`, this function cannot be re-used to render a standalone component, which is what React intends to solve. Can you refactor the code so that the entire table is rendered in one function?

Once you do that, you can also try going one step further by making this table into a React component such that you can simply write `<AppointmentData ...>` to render it instead of using `{AppointmentData()}`. This is the ideal way to use React because you should be thinking in terms of components instead of functions that generate html code.

Reply:
I just solved it by this method, and it seems so cool, since it even does not require a `prop` due to `useQuery` included.